packages/react-app/src/App.jsx

```diff
@@ -42,7 +42,7 @@ const web3Modal = new Web3Modal({
   },
 });
 
-function App() {
+function App({ setLocale }) {
```

Comment:
we can remove `{ setLocale }` since its not being used

Reply:
oh yes, I forgot it there. Will remove!